### PR TITLE
New package: nitrokey-app

### PR DESCRIPTION
**Pull request by renatoaguiar**

### CODE REVIEW

File: srcpkgs/nitrokey-app/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+maintainer="Renato Aguiar <contact@renatoaguiar.org>"
+license="GPL-3"
+homepage="https://github.com/Nitrokey/nitrokey-app"
+distfiles="https://github.com/Nitrokey/${pkgname}/archive/v${version}.tar.gz"
+checksum=7d907c722aa621e7510e488358c0ddb533f433aee94a75865729c76e6e5071e7
+
+post_extract() {
+	sed -i 's|/etc/udev/rules.d|/usr/lib/udev/rules.d|g' CMakeLists.txt
+	sed -i 's|/etc/bash_completion.d|/usr/share/bash-completion/completions|g' \
+	    CMakeLists.txt
```
Comment (Vaelatern) [inline srcpkgs/nitrokey-app/template]:
```
sed -i CMakeLists.txt \
-e 's|/etc/udev/rules.d|/usr/lib/udev/rules.d|g' \
-e 's|/etc/bash_completion.d|/usr/share/bash-completion/completions|g'
```

File: srcpkgs/nitrokey-app/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+version=0.2
+revision=1
+build_style=cmake
+hostmakedepends="cmake pkg-config"
+makedepends="qt5-devel libusb-devel"
+short_desc="Nitrokey's Application"
+maintainer="Renato Aguiar <contact@renatoaguiar.org>"
+license="GPL-3"
+homepage="https://github.com/Nitrokey/nitrokey-app"
+distfiles="https://github.com/Nitrokey/${pkgname}/archive/v${version}.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/nitrokey-app/template]: Might as well use `${homepage}` here.

File: srcpkgs/nitrokey-app/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'nitrokey-app'
+pkgname=nitrokey-app
+version=0.2
+revision=1
+build_style=cmake
+hostmakedepends="cmake pkg-config"
+makedepends="qt5-devel libusb-devel"
+short_desc="Nitrokey's Application"
+maintainer="Renato Aguiar <contact@renatoaguiar.org>"
+license="GPL-3"
```
Comment (Vaelatern) [inline srcpkgs/nitrokey-app/template]: Significant parts of it are Apache-2.0. `license="GPL-3, Apache-2.0"` unless you have reason to say it's only GPL.

File: srcpkgs/nitrokey-app/template [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'nitrokey-app'
+pkgname=nitrokey-app
+version=0.2
+revision=1
+build_style=cmake
+hostmakedepends="cmake pkg-config"
+makedepends="qt5-devel libusb-devel"
+short_desc="Nitrokey's Application"
```
Comment (Duncaen) [inline srcpkgs/nitrokey-app/template]: A better short description would be nice.

File: srcpkgs/nitrokey-app/template [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'nitrokey-app'
+pkgname=nitrokey-app
+version=0.2
+revision=1
+build_style=cmake
+hostmakedepends="cmake pkg-config"
```
Comment (Vaelatern) [inline srcpkgs/nitrokey-app/template]: As of last night, you no longer need to include `cmake` on this list because it is `build_style=cmake`.

**Pull request merged**